inspirehep/modules/search/views.py

```diff
@@ -69,6 +69,12 @@ def search():
                            **ctx)
 
 
+@blueprint.route("/ping")
+def ping():
+    """Ping for load balancer."""
+    return "OK", 200
```

Comment:
Whoops, you're right about the `url_prefix`. On the other hand, Isn't this guy already defined? https://github.com/inspirehep/inspire-next/blob/644530717f503b4913f58336d7b5246088fbac24/inspirehep/modules/theme/views.py#L172-L174

Maybe we just need to add an explicit 200 to that?